lib/Test2/Harness/UI/Schema/Overlay/Run.pm

```diff
@@ -93,17 +93,7 @@ sub expanded_coverages {
     my $self = shift;
     my ($query) = @_;
 
-    my $pick_me = {run_id => $self->run_id};
-
-    if ($query) {
-        $query = {'-and' => [$query, $pick_me]};
-    }
-    else {
-        $query = $pick_me;
-    }
-
-    my $schema = $self->result_source->schema;
-    $schema->resultset('Coverage')->search(
+    $self->coverages->search(
```

Comment:
I do not understand how this change works, can you explain it?

Reply:
The [`has_many`](https://metacpan.org/pod/DBIx::Class::Relationship#has_many) declaration in [`Schema::Run`](https://github.com/ilmari/Test2-Harness-UI/blob/schema-fixes/lib/Test2/Harness/UI/Schema/PostgreSQL/Run.pm#L100-L105) creates an accessor that's equivalent to `->related_resultset($name)`.